src/ml_board/frontend/Dashboard/src/components/experimentPage/ExperimentDetails.tsx/ExperimentDetails.tsx

```diff
@@ -0,0 +1,127 @@
+                        cardTitle='Overview'
+                        contentObj={
+                            {   
+                                "Experiment ID": selectedExperiment.experiment_id,
+                                "Job Type": 
+                                selectedExperiment.job_type ?
+                                selectedExperiment.job_type
+                                :
+                                "--",
+                                "Device":
+                                selectedExperiment.device ? 
+                                selectedExperiment.device
+                                :
+                                "--"
+                            }
+                        }
+                    />
+                </Grid>
+                <Grid item={true} xs={12} sm={12} md={4}>
+                    <CardDetails 
+                        cardTitle='Times'
+                        contentObj={
+                            {   
+                                "Start": selectedExperiment.starting_time && selectedExperiment.starting_time !== -1 ?
+                                moment(new Date(selectedExperiment.starting_time*1000)).format("YYYY-MM-DD hh:MM:SS")
+                                :
+                                "--",
+                                "End": selectedExperiment.finishing_time && selectedExperiment.finishing_time !== -1 ?
+                                moment(new Date(selectedExperiment.finishing_time*1000)).format("YYYY-MM-DD hh:MM:SS")
+                                :
+                                "--"
+                            }
+                        }
+                    />
+                </Grid>
+                <Grid item={true} xs={12} sm={12} md={4}>
+                    <CardDetails 
+                        cardTitle='State'
+                        contentObj={
+                            {   
+                                "Job Status": 
+                                selectedExperiment.hasOwnProperty("error") && selectedExperiment.error !== null ?
+                                "FAILED"
+                                :
+                                selectedExperiment.job_status === "DONE" ?
+                                "COMPLETED"
+                                :
+                                selectedExperiment.job_status === "INIT" ?
+                                "INITIALIZING"
+                                :
+                                selectedExperiment.job_status ?
+                                selectedExperiment.job_status
+                                :
+                                "--",
+                                "Model Status": 
+                                selectedExperiment.model_status ?
+                                selectedExperiment.model_status 
+                                :
+                                "--"
+                            }
```

Comment:
is this formatted correctly? It's a bit hard to read. The same formatting is used in many other places.

Reply:
I reformatted that file hopefully for better readability, check it again?!

Reply:
I have created functions for conditional rendering, which had improved the readability

Reply:
yeah but things like are impossible to read 
```
 "Job Status": 
                                selectedExperiment.hasOwnProperty("error") && selectedExperiment.error !== null ?
                                "FAILED"
                                :
                                selectedExperiment.job_status === "DONE" ?
                                "COMPLETED"
                                :
                                selectedExperiment.job_status === "INIT" ?
                                "INITIALIZING"
                                :
                                selectedExperiment.job_status ?
                                selectedExperiment.job_status
                                :
                                "--",
                                "Model Status": 
                                selectedExperiment.model_status ?
                                selectedExperiment.model_status 
                                :
```